website/content/en/preview/concepts/provisioners.md

```diff
@@ -258,7 +258,7 @@ For more information on weighting Provisioners, see the [Weighting Provisioners
 ## spec.kubeletConfiguration
 
 Karpenter provides the ability to specify a few additional Kubelet args. These are all optional and provide support for
-additional customization and use cases. Adjust these only if you know you need to do so. For more details on kubelet configuration arguments, [see the KubeletConfiguration API specification docs](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/).
+additional customization and use cases. Adjust these only if you know you need to do so. For more details on kubelet configuration arguments, [see the KubeletConfiguration API specification docs](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/). The implemented fields are a subset of the full list of upstream kubelet configuration arguments. Please cut an issue if you'd like to see another field implemented.
```

Comment:
Should we be more explicit about why we have chosen certain fields? e.g. They affect our ability to do scheduling simulations so we need them in both places as opposed to asking the user to pass them through userData?